diff --git a/flashcp.c b/flashcp.c
index f76a8c2..4cea0b6 100644
--- a/flashcp.c
+++ b/flashcp.c
@@ -45,7 +45,7 @@
 #include <syslog.h>
 #include <linux/reboot.h>
 
-typedef int bool;
+typedef int Bool;
 #define true 1
 #define false 0
 
@@ -96,7 +96,7 @@ static void log_printf (int level,const char *fmt, ...)
 	fflush (fp);
 }
 
-static void showusage(bool error)
+static void showusage(Bool error)
 {
 	int level = error ? LOG_ERROR : LOG_NORMAL;
 
@@ -139,7 +139,7 @@ static int safe_open (const char *pathname,int flags)
 	return (fd);
 }
 
-static int safe_read (int fd,const char *filename,void *buf,size_t count,bool verbose)
+static int safe_read (int fd,const char *filename,void *buf,size_t count,Bool verbose)
 {
 	ssize_t result;
 
